meetings/2025-06-17.md

```diff
@@ -18,3 +14,53 @@ Joining info:
 4. [Using IRC to generate Minutes](https://github.com/w3c-cg/webdocs/issues/13)
 
 ## Notes
+
+Introductions.
+
+## TPAC
+
+- Lola: Friday is the deadline for proposals. Regarding https://github.com/w3c-cg/webdocs/issues/12 - should we have drop-in sessions for WGs, or an MDN contribution workshop?
+- Estelle: both!
+- Explainers: help people understand how to write good epxlainers
+- Estelle: should have: (1) CG session for this grouo, face to face and (2) external session. Figure out  how to get tech writers and spec authors to get together (technical networking). For explainers, if this is the rioght group then yes, but it should go soemnwhere
+- Lola: TAG owns explainer process.
+- Estelle: can Lola ask TAG if they are going to have an explainer session? Otherwise we will.
+- Lola: probably they won't because we know that explainers aren't going anywhere.
+- Lola might not be at TPAC, if she is, TAG and WebDX will take up much of her time.
+- Will: workshop with Web Components people (and anyone else who wants to work with us on their docs)
+- Florian: good to have some concrete topics, e.g. meeting with specific WGs like Web Components. Also work on some strategic stuff for this CG. Talk about "docs as conversations" (LLMs) at TPAC and what this means for docs.
+- FRançois: haven't heard back from Web Components about whether they want to meet, usually they want to meet in breakouts. Trying to encourage them to meet in part of the main schedule. Instead of booking slots, andother approach would be to get on WG agenda and go to their meeting. Might get more WG people then. Risk of hosting our own sessions is that only docs CG people would attend.
```

Comment:
```suggestion
- François: haven't heard back from Web Components about whether they want to meet, usually they want to meet in breakouts. Trying to encourage them to meet in part of the main schedule. Instead of booking slots, and other approach would be to get on WG agenda and go to their meeting. Might get more WG people then. Risk of hosting our own sessions is that only docs CG people would attend.
```